classes/DepositPoint.inc.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/**
+ * @file plugins/generic/sword/classes/DepositPoint.inc.php
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2003-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class DepositPoint
+ * @ingroup plugins_generic_sword_classes
+ *
+ * @brief Deposit point entity.
+ */
+
+class DepositPoint extends DataObject {
+	/**
+	 * Get the ID of the deposit point.
+	 * @return int
+	 */
+	public function getId() {
```

Comment:
`DataObject`s now include a default `getId`/`setId` implementation that should be enough. (The key will change from `depositPointId` to `id`, IIRC, but that shouldn't cause side-effects if all accesses are done via `getId`/`setId` as they should be.)

Reply:
At least 9 years ago: https://github.com/pkp/pkp-lib/blame/master/classes/core/DataObject.inc.php#L169

Reply:
Updated.